Source/ProgressionSystemRuntime/Public/Data/PSDataAsset.h

```diff
@@ -143,4 +148,8 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSDataAsset : public UMyPrimaryDataAsset
 	/** When Instant character switch setting enabled fade animation will not be played */
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "C++", AdvancedDisplay, meta = (BlueprintProtected, DisplayName = "Instant Character Switch Tag"))
 	FSettingTag InstantCharacterSwitchTagInternal = FSettingTag::EmptySettingTag;
+
+	/** A padding setting for the Menu Widget displayed in the end game state/menu */
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "C++", AdvancedDisplay, meta = (BlueprintProtected, DisplayName = "Padding for Stars Menu Widget"))
+	FMargin MenuWidgetPaddingInternal = FMargin(0, 0, 0, 0);
```

Comment:
You dont need Padding in data asset anymore, please remove `MenuWidgetPaddingInternal` property and `GetMenuWidgetPadding()` function

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/43/commits/916f3a869934724326a5d17adff9b09e7f6a3656